Gemfile.base

```diff
@@ -36,8 +36,8 @@ end
 group :development do
   gem 'sshkit'
   gem 'source2swagger', git: 'https://github.com/3scale/source2swagger', branch: 'backend'
-  gem 'pry',      '~> 0.11.3'
-  gem 'pry-doc',  '~> 0.11.1'
+  gem 'pry',      '~> 0.14'
+  gem 'pry-doc',  '~> 1.1'
```

Comment:
`~> 1` should be enough here if pry-doc follows semver (that is not the case with `0.x` versions).

Reply:
1.0.0 doesn't support 2.7, so the minimum is 1.1.